xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1210 @@
+   </screen>
+     </step>
+     <step>
+      <para>
+       Create basic authentication credentials for the Docker registry.
+      </para>
+      <para>
+       Replace <literal>USERNAME</literal> and <literal>PASSWORD</literal> with
+       proper credentials of your choosing.
+      </para>
+<screen>&prompt.user;<command>mkdir auth</command>
+&prompt.user;<command>docker run --entrypoint htpasswd sles12/registry:2.6.2 -Bbn <replaceable>USERNAME PASSWORD</replaceable> > auth/htpasswd</command>
+ </screen>
+     </step>
+     <step>
+      <para>
+       Create the <filename>/etc/docker/registry/config.secure.yml</filename>
+       configuration file for Docker registry.
+      </para>
+      <para>
+       Replace <filename>/certs/domain.crt</filename> and
+       <filename>/certs/domain.key</filename> with the appropriate certificate
+       files for your organization.
+      </para>
+<screen>
+version: 0.1
+log:
+  fields:
+    service: registry
+auth:
+  htpasswd:
+    realm: basic-realm
+    path: /auth/htpasswd
+storage:
+  cache:
+    blobdescriptor: inmemory
+  filesystem:
+    rootdirectory: /var/lib/registry
+http:
+  addr: 0.0.0.0:5000
+  headers:
+    X-Content-Type-Options: [nosniff]
+  tls:
+    certificate: /certs/domain.crt
+    key: /certs/domain.key
+health:
+  storagedriver:
+    enabled: true
+    interval: 10s
+threshold: 3
+</screen>
+      <para>
+       For more details on the configuration, refer to:
+       <link xlink:href="https://docs.docker.com/registry/configuration/">Docker
+       Registry: Configuration</link>
+      </para>
+     </step>
+     <step>
+      <para>
+       Start the docker service and enable it at boot time:
+      </para>
+<screen>&prompt.root;<command>systemctl enable --now docker.service</command>
+         </screen>
+     </step>
+     <step>
+      <para>
+       Start the registry container.
+      </para>
+<screen>&prompt.user;<command>cd /etc/docker/registry</command>
+&prompt.user;<command>docker run -d -p 5000:5000 --restart=always --name registry \
+-v `pwd`<replaceable>/config.secure.yml</replaceable>:/etc/docker/registry/config.yml \
+-v `pwd`/auth:/auth \
+-v `pwd`/certs:/certs \
+-v <replaceable>/registry</replaceable>:/var/lib/registry sles12/registry:2.6.2</command>
+       </screen>
+     </step>
+    </procedure>
+
+    <para>
+     Now we will move on to the internal server in the airgapped network and
+     perform a nearly identical procedure there.
+    </para>
+
+    <procedure>
+     <title>Set Up Internal Mirror</title>
+     <step>
+      <para>
+       SSH into the internal mirror server.
+      </para>
+     </step>
+     <step>
+      <para>
+       Connect the trusted storage to the internal mirror and copy the registry
+       container image to local filesystem.
+      </para>
+<screen>&prompt.user;<command>cp /mnt/storage/registry.tar /etc/docker/registry/registry.tar</command>
```

Comment:
There's no need to copy the registry to this location. The user can do a simple docker load pointing straight to the USB disk.

Reply:
`docker load` loads the image from the usb stick into the running docker daemon. There's no need to copy the tarball from the usb to the computer hard drive and then do the `docker load`. You are going to save one step